### DISCUSSION TITLE
[Non-NASA] Control doesn't working

### DISCUSSION THREAD

**Discussion by cavermanus** [omerfaruk-aran/esphome_samsung_hvac_bus#150] [Questions]
Hello, I've a MH040FXEA2 outdoor with two MH026FWEA indoor units. I made connection with mentioned M5STACK ATOM Lite and ATOM RS485 Base on F1-F2 lines. The integration looks OK, I look the power states, modes, target temperatures in HA. Unfortunatelly the controlling does't work.

I did monitoring on the ATOM TX and RX pins. On RX the data is coming continously and I think there isn't enough room to inject the commands from ATOM. When the ATOM attempted to send data, the data is disintegrated on the RS485 lines.

Has anyone had a similar experience?
Thx!

**Comment by omerfaruk-aran** [comment]
Hello! 🌟

It’s been some time since you posted your question. Were you able to find a solution or make any progress with your setup?

From what you described, it seems like there might be a timing or communication conflict on the RS485 bus. This is sometimes caused by continuous data flow on the RX line, which could be preventing the injection of new commands.

If you haven't solved it yet, let me know, and we can dig deeper into potential causes, like baud rate adjustments, wiring checks, or command timing improvements.

Looking forward to hearing from you!

**Reply by zerp0** [reply]
i have the same problem, can i test some things to get this resolved? lots of rx , but no control functions
